lib/matplotlib/figure.py

```diff
@@ -862,7 +862,13 @@ def set_size_inches(self, w, h=None, forward=True):
 
         See Also
         --------
-        matplotlib.Figure.get_size_inches
+        matplotlib.figure.Figure.get_size_inches
+        matplotlib.figure.Figure.set_figwidth
+        matplotlib.figure.Figure.set_figheight
+
+        Notes
+        -----
+        To transform from pixels to inches divide by `Figure.dpi`.
```

Comment:
I'm not sure we should be pointing this out explicitly.  I think that 90% of the time we don't want people to think in pixels.  Also, when saving, `Figure.dpi` is just a default anyway. I'd prefer this discussion happen in `savefig`.

Reply:
I assume pixels is more or less what people are actually thinking in most of the time, because that's their target unit if they draw to screen or png. At least, that's my experience. - Also it's the unit GUI frameworks, websites (often), image programs etc use. Maybe Graphics designers and TeX people don't.

Reply:
I think even the web is moving towards device independence.

Reply:
It's just a note, and people are looking for this info:
https://stackoverflow.com/questions/29702424/how-to-get-matplotlib-figure-size